peps/tests/test_commands.py

```diff
@@ -6,16 +6,14 @@
 from django.conf import settings
 from django.core import serializers
 from django.core.management import call_command
-from django.core.exceptions import ImproperlyConfigured
 
 import responses
 
 from pages.models import Image
 
-from . import FAKE_PEP_REPO
+from . import FAKE_PEP_ARTIFACT
 
 
-@responses.activate
 class PEPManagementCommandTests(TestCase):
 
     def setUp(self):
```

Comment:
I think we can use `setUpTestData` here. No need to populate this after each test. I don't know how the `responses.add()` call would work, though.

Reply:
👌 Fixed